### PR TITLE
added collection workflow TCs

### PR DESCRIPTION
**Pull request by yatheendra-egov**

### CODE REVIEW

File: src/test/java/com/egov/business-services/preTests/propertyTaxAssessmentPretest.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    """
+  * print assessmentRequest
+  Given url createAssessment
+  And params assessmentParams
+  And request assessmentRequest
+  When method post
+  Then status 201
+  * print response
+  # And match response.ResponseInfo.status == 'successful'
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/business-services/preTests/propertyTaxAssessmentPretest.feature:29]: Can we remove this comment?

File: src/test/java/com/egov/core-services/eGovUser/passwordNologinUpdate.feature [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+@Update_NoLogin_Password_InValidUserName_08  @negative   @userPasswordNoLogin @user
+Scenario: To validate error message when userName is missing
+        * call read('../pretests/updatePasswordNoLoginPretest.feature@inValidUserNameOnUpdatePassword')
+        * match updatedPasswordWithOutLogin.Errors[0].code == errorMessage.errormessages.userNotFoundError
+
+@Update_NoLogin_Password_InValidtenantId_09  @negative   @userPasswordNoLogin @user
+Scenario: To validate error message when tenantId is missing
+        * call read('../pretests/updatePasswordNoLoginPretest.feature@inValidTenantIdOnUpdatePassword')
+        * match updatedPasswordWithOutLogin.Errors[0].code == errorMessage.errormessages.userNotFoundError
+
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/core-services/eGovUser/passwordNologinUpdate.feature:45]: @krishnendu-eGov please remove empty lines.

File: src/test/java/com/egov/core-services/eGovUser/passwordNologinUpdate.feature [hunk 3]
```diff
@@ -0,0 +1,49 @@
+Feature: Update user
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/core-services/eGovUser/passwordNologinUpdate.feature:1]: @krishnendu-eGov rename this as seeing same Update user in other files as well

File: src/test/java/com/egov/core-services/eGovUser/passwordUpdate.feature [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        * call read('../pretests/updateUserPasswordPretest.feature@validPassword')
+        * assert updatedPasswordResponseBody.responseInfo.status == 200
+
+@Update_Password_ValidExistingPassword_InvalidNewPassword_02 @negative @userPassword @user
+Scenario: To verify the invalid length password error
+   * call read('../pretests/updateUserPasswordPretest.feature@inValidNewPassword')
+   * assert updatedPasswordResponseBody.Errors[0].code == errorMessage.errormessages.invalidPasswordLngthErrCode
+   * assert updatedPasswordResponseBody.Errors[0].message == errorMessage.errormessages.invalidPasswordLengthError
+   
+
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/core-services/eGovUser/passwordUpdate.feature:21]: @krishnendu-eGov please remove extra empty lines from every file and keep it uniform as single line

File: src/test/java/com/egov/core-services/pretests/updateUserPasswordPretest.feature [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    And request updatedUserPassword
+     * print updatedUserPassword
+    When method post
+    Then status 400
+    And  def updatedPasswordResponseBody = response
+     * print updatedPasswordResponseBody
+
+  @inValidExistingPassword
+  Scenario: Update existing password with invalid existing password
+   * configure headers = read('classpath:websCommonHeaders.js')
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/core-services/pretests/updateUserPasswordPretest.feature:44]: @krishnendu-eGov - Why are we using headers in all pre tests? We can keep it in background.

File: src/test/java/com/egov/core-services/pretests/updateUserPretest.feature [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+     * call read('../pretests/authenticationToken.feature')
+     * def updatedUserProfile = read('../requestPayload/user/update/updateUser.json')
+     * def userProfileData =    read('../constants/user.yaml')
+
+  @validParameters
+  Scenario: Update existing user profile with valid parameters
+   * set updatedUserProfile.user.name = userProfileData.validProfileData.userName
+   * set updatedUserProfile.user.emailId = userProfileData.validProfileData.emailId
+   * set updatedUserProfile.user.correspondenceCity = userProfileData.validProfileData.city
+   * configure headers = read('classpath:websCommonHeaders.js')
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/core-services/pretests/updateUserPretest.feature:20]: please use it in background

File: src/test/java/com/egov/core-services/requestPayload/user/update/updateUser.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+        "permanentPinCode": null,
+        "correspondenceAddress": "#(correspondenceAddress)",
+        "correspondenceCity": "#(correspondenceCity)",
+        "correspondencePinCode": null,
+        "active": true,
+        "locale": null,
+        "type": "EMPLOYEE",
+        "accountLocked": false,
+        "accountLockedDate": 0,
+        "fatherOrHusbandName": "TEST",
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/core-services/requestPayload/user/update/updateUser.json:27]: json should not have any hard coded values.

File: src/test/java/com/egov/core-services/requestPayload/user/updatePassword/updatePassword.json [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{
+  "RequestInfo": {
+    "authToken": "#(authToken)"
+  },
+  "existingPassword": "#(existingPassword)",
+  "newPassword": "#(newPassword)",
+  "tenantId": "#(newPassword)",
+  "username": "#(userName)",
+  "type": "EMPLOYEE"
```
Comment (pundaleek-egov) [inline src/test/java/com/egov/core-services/requestPayload/user/updatePassword/updatePassword.json:9]: you can read it from yaml file

**Comment by krishnendu-eGov** [comment]
@yatheendra-egov and @pundaleek-egov I have addressed all of the review comments. Please check it now.

**Review by pundaleek-egov — APPROVED** [review]

**Pull request merged**